### PR TITLE
Sub sup buttons

### PR DESCRIPTION
**Pull request by tarjei**
## Checklist

- [x] Fix any eslint errors that occur
- [x] Update change-log for every plugin you touch
- [x] Add/Update docs if you add/change functionality
- [x] Enable "Allow edits from maintainers" for this PR

## Use-case/Problem
Fix for #882 : This PR adds two new buttons. 

## Implementation
The implementation follows the other inline button implementations. 

## Demo
I've added the buttons to the storybook and also updated the storybook config to include the StaticToolbar

### CODE REVIEW

**Comment by juliankrispel** [comment]
sweet pr @tarjei thanks

File: stories/StaticToolbar/CustomToolbarEditor/index.js [hunk 1]
```diff
@@ -75,12 +77,14 @@ const toolbarPlugin = createToolbarPlugin({
...
-    CodeBlockButton
+    CodeBlockButton,
+    SubButton,
+    SupButton
   ]
 });
 const { Toolbar } = toolbarPlugin;
 const plugins = [toolbarPlugin];
-const text = 'In this editor a toolbar shows up once you select part of the text …';
+const text = 'Remember to place the <Toolbar> component bellow the Editor component …';
```
Comment (juliankrispel) [inline stories/StaticToolbar/CustomToolbarEditor/index.js]: I like that!

File: draft-js-buttons/src/utils/createOnClickButton.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+            className={className}
+            onClick={this.activate}
+            type="button"
+            children={children}
+          />
+        </div>
+      );
+    }
+  }
+);
```
Comment (juliankrispel) [inline draft-js-buttons/src/utils/createOnClickButton.js]: this doesn't seem to be used anywhere why is this in here?

File: draft-js-buttons/src/index.js [hunk 3]
```diff
@@ -15,12 +17,18 @@ import AlignBlockDefaultButton from './components/AlignBlockDefaultButton';
...
 import AlignBlockLeftButton from './components/AlignBlockLeftButton';
 import AlignBlockRightButton from './components/AlignBlockRightButton';
+import SubButton from './components/SubButton';
+import SupButton from './components/SupButton';
 
 export {
   createBlockStyleButton,
   createInlineStyleButton,
+  createBlockAlignmentButton,
+  createOnClickButton,
```
Comment (juliankrispel) [inline draft-js-buttons/src/index.js]: why are these two being exported here? Please remove

File: stories/index.js [hunk 4]
```diff
@@ -129,3 +134,8 @@ storiesOf('Video Plugin')
   .add('Editor with Video Plugin', () => <SimpleVideoEditor />)
   .add('Editor with custom themed Video Plugin', () => <CustomVideoEditor />)
   .add('Editor with Video Plugin and Add Video Button', () => <CustomAddVideoEditor />);
+
+storiesOf('StaticToolbar Plugin')
+  .add('CustomToolbarEditor', () => <CustomToolbarEditor />)
+  .add('Simple toolbar editor', () => <SimpleToolbarEditor />)
+  .add('ThemedToolbarEditor', () => <ThemedToolbarEditor />);
```
Comment (juliankrispel) [inline stories/index.js]: Mind adding it to the InlineToolbar as well please

**Comment by juliankrispel** [comment]
@tarjei few comments but generally looking good thanks

File: draft-js-buttons/src/index.js (hunk 3 above)
Comment (tarjei) [reply draft-js-buttons/src/index.js]: creatOnClick button is used to create buttons with custom click events that are outside the "standard" toolbar implementations. Example usage: 

```
import React, { Component } from 'react'
import PropTypes from 'prop-types'
import insertTeXBlock from 'draft-js-katex-plugin/lib/modifiers/insertTeXBlock'

import asciimath2latex from 'asciimath-to-latex'

import createOnClickButton from './createOnClickButton'

export default createOnClickButton({
  onClick: ({ getEditorState, setEditorState }) => {
    setEditorState(insertTeXBlock(getEditorState(), asciimath2latex, 'int(x-2)', false))
  }
})
import React, { Component } from 'react'
import PropTypes from 'prop-types'
import insertTeXBlock from 'draft-js-katex-plugin/lib/modifiers/insertTeXBlock'

import asciimath2latex from 'asciimath-to-latex'

import createOnClickButton from './createOnClickButton'

export default createOnClickButton({
  onClick: ({ getEditorState, setEditorState }) => {
    setEditorState(insertTeXBlock(getEditorState(), asciimath2latex, 'int(x-2)', false))
  }
})
```

File: draft-js-buttons/src/utils/createOnClickButton.js (hunk 2 above)
Comment (tarjei) [reply draft-js-buttons/src/utils/createOnClickButton.js]: See comment above.

File: stories/index.js (hunk 4 above)
Comment (tarjei) [reply stories/index.js]: I'll see what I can do. Is it an option for you to just edit the PR to include this?

File: draft-js-buttons/src/index.js (hunk 3 above)
Comment (juliankrispel) [reply draft-js-buttons/src/index.js]: hey @tarjej. Thanks! I think it's a good idea to add Sup/Sub buttons, but am not sure yet whether adding a generic button is a good idea and if so what the api of such a button should be. In any case it should be considered a different pr. So please remove it from this one :+1:

File: stories/index.js (hunk 4 above)
Comment (juliankrispel) [reply stories/index.js]: If I have to I will but if you can help it'd be great since maintaining the repository is already quite a lot of work :heart:

**Pull request merged**